### ISSUE TITLE
Some icons not showing

### ISSUE THREAD

**Issue by KevinLamza** [SylEleuth/gruvbox-plus-icon-pack#75]
Hello,

I'm running Ubuntu 22.04 and some of the icons are not showing. I tried both downloading the folder and also cloning from GitHub. Both cases result in the same.
![Screenshot from 2024-08-21 02-30-48](https://github.com/user-attachments/assets/143796d2-30c2-4c61-85a4-65657f345209)

**Comment by SylEleuth** [comment]
How did You install `Discord`? Because I've just tested it on `Debian` (had problems to run my `Ubuntu` vm) from official and `flatpak` repos and icon displays with no issues. `Snap` on the other hand doesn't respect icon from the pack. I remember that some time ago I tried to find a solution but eventually gave up. Probably it's hardcoded somewhere and unfortunately I can't do anything about it. I don't know about others but if You used Snap it will be the same.
